commands/plugin_get.go

```diff
@@ -31,9 +33,11 @@ func init() {
 		RunE:    runPluginGetCmd,
 	}
 
+	clientArch, operatingSystem := env.GetClientArch()
+
 	pluginGetCmd.Flags().StringVar(&pluginRegistry, "registry", "ghcr.io/openfaasltd", "The registry to pull the plugin from")
-	pluginGetCmd.Flags().StringVar(&arch, "arch", "amd64", "The architecture to pull the plugin for")
-	pluginGetCmd.Flags().StringVar(&clientOS, "os", "linux", "The OS to pull the plugin for")
+	pluginGetCmd.Flags().StringVar(&arch, "arch", clientArch, "The architecture to pull the plugin for")
```

Comment:
Now that we determine these, they should be empty "" blank, and a note that says "give a value or leave blank for auto-detection"

Reply:
The default is now blank and I a note was added.